docs/deploying.md

```diff
@@ -81,13 +83,13 @@ order of operations is:
 Build new services by using the script at `./scripts/build`, specifying the
 following values:
 
--   `PROJECT_ID` (required) - your Google Cloud project ID.
```

Comment:
This indentation is intentional, because then nested fields (e.g. codeblocks inside a list) align correctly.

Reply:
(everywhere)

Reply:
these are robotic changes - there is nobody on the other end to adjust

I can ask matt to turn off the markdown formatter.